### PR TITLE
Improve the user settings interface

### PR DESCRIPTION
**Pull request by dkanada**
The first commit removes the section below and replaces it with some buttons, the old process was awkward to get through. I'll add a few more commits that change the headers and remove old strings when I have time.

remove

### CODE REVIEW

**Comment by JustAMan** [comment]
@dkanada can you show "before" and "after"? I cannot grasp the pic you show, is it the old state or the proposed?

**Comment by dkanada** [comment]
The photo in my last comment is before the change, I just replaced it with a single button to add the photo. It looks like the `Scan All Libraries` button in the dashboard.

before
after

File: src/strings/de.json [hunk 1]
```diff
@@ -1296,7 +1290,7 @@
     "ButtonTrailer": "Trailer",
     "ButtonWebsite": "Webseite",
     "ChangingMetadataImageSettingsNewContent": "Änderungen an Metadaten- und Artwork-Einstellungen betreffen nur der Bibliothek neu hinzugefügte Inhalte. Um diese Änderungen auf existierende Medien anzuwenden müssen die Metadaten manuell aktualisiert werden.",
-    "CopyStreamURL": "Kopiere Streaminglink",
+    "CopyStreamURL": "Stream URL kopieren",
     "CopyStreamURLSuccess": "Link erfolgreich kopiert.",
```
Comment (thornbill) [inline src/strings/de.json]: Is this an intentional change to the translation?

Comment (dkanada) [reply src/strings/de.json]: Apparently someone added the key twice in this file, so my script replaced this instance with a value from the same key at the end of the file.

**Pull request merged**